nemo/collections/nlp/parts/megatron_trainer_builder.py

```diff
@@ -53,6 +53,7 @@ def _training_strategy(self) -> NLPDDPStrategy:
             no_ddp_communication_hook=True,
             gradient_as_bucket_view=self.cfg.model.gradient_as_bucket_view,
             find_unused_parameters=False,
+            sharp=cfg.model.get('sharp', False),
```

Comment:
@erhoo82 probably a typo. It should be `self.cfg.model.get` right ?

Reply:
This breaks the running of this PR